File: DevHub_V1/app/src/main/java/com/devhub/mirza/devhub_v1/UIObjects/LoginResultDTO.java
package com.devhub.mirza.devhub_v1.UIObjects;

import java.io.Serializable;

public class LoginResultDTO implements Serializable {
    private AuthenticationTokens token;
    private Users user;
    public AuthenticationTokens getToken() {
        return token;
    }

    public void setToken(AuthenticationTokens token) {
        this.token = token;
    }

    public Users getUser() {
        return user;
    }

    public void setUser(Users user) {
        this.user = user;
    }
}

File: DevHub_API/DevHub_API/DTOs/LoginResultDTO.cs
using DevHub_API.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace DevHub_API.DTOs
{
    public class LoginResultDTO
    {
        public AuthenticationTokens token { get; set; }
        public Users user { get; set; }
    }
}